helm_deploy/laa-estimate-eligibility/templates/_envs.tpl

```diff
@@ -110,11 +112,11 @@ env:
       secretKeyRef:
         name: kube-secrets
         key: blazer-database-password
-  - name: FEATURE_FLAGS_PASSWORD
+  - name: BASIC_AUTH_PASSWORD
     valueFrom:
       secretKeyRef:
         name: kube-secrets
-        key: feature-flags-password
+        key: basic-auth-password
```

Comment:
question - this stuff is going "over my head". Am I to understand that here we are updating where we now point to where our password values are kept???

Reply:
So there is a box of secrets, and it's called `kube-secrets`, and we used to pull a secret called `feature-flags-password` out of it and use it to set an environment variable called `FEATURE_FLAGS_PASSWORD`. Now we don't need that any more, so we've deleted it. We now pull a secret called `basic-auth-password` out of the box of secrets, and use it to populate an environment variable called `BASIC_AUTH_PASSWORD`.